### ISSUE TITLE
freerouting 1.6 can't load any dsn file from Kicad on Windows 7 64bit

### ISSUE THREAD

**Issue by cdhigh** [freerouting/freerouting#118]
Freerouting v1.6 (msi installer and jar package) can't load any dsn file from Kicad on Windows 7 64bit.

Both v1.5 and v1.4.4 are working great.

First of all, thank you for taking the time to report an issue.

To make our work easier, please provide as many details as you can:
* [ ] What version of freerouting are you using?
tested with freerouting 1.6 msi and jar, the same problem

* [ ] On what platform did you run it? Windows, Linux, MacOS? 32-bit or 64-bit?
Windows 7 64bit zh-cn

* [ ] What type of file are you working with? Can you attach the actual (DSN) file?
Dsn exported from Kicad 5.1.12, I tested with several dsn, the same problem
[testauto.zip](https://github.com/freerouting/freerouting/files/8853553/testauto.zip)

* [ ] Did you use any command line arguments?
Opened with GUI and executed with argument -de testauto.dsn, same problem

* [ ] Can you post a screenshot of the problem?

First, display this exception info.
![freerouting_1_6_exception](https://user-images.githubusercontent.com/2580077/172392459-23dc612f-e567-449c-8b41-0799bec8a0d6.png)

after, stuck on this dialog.
![stuck](https://user-images.githubusercontent.com/2580077/172392498-58cda942-4b65-4382-a914-f34369fbd4cd.png)

* [ ] Can you include the latest, related log entries from the .log file?
```
2022-06-07 10:31:51,177 main WARN JMX runtime input lookup class is not availabl
e because this JRE does not support JMX. JMX lookups will not be available, cont
inuing configuration. Ignoring java.lang.IllegalArgumentException: java.lang.NoC
lassDefFoundError: Could not initialize class org.apache.logging.log4j.core.look
up.JmxRuntimeInputArgumentsLookup
2022-06-07 10:31:51,333 main ERROR Could not reconfigure JMX java.lang.NoClassDe
fFoundError: javax/management/MBeanRegistrationException
        at org.apache.logging.log4j.core.LoggerContext.setConfiguration(LoggerCo
ntext.java:632)
        at org.apache.logging.log4j.core.LoggerContext.reconfigure(LoggerContext
.java:694)
        at org.apache.logging.log4j.core.LoggerContext.reconfigure(LoggerContext
.java:711)
        at org.apache.logging.log4j.core.LoggerContext.start(LoggerContext.java:
253)
        at org.apache.logging.log4j.core.impl.Log4jContextFactory.getContext(Log
4jContextFactory.java:155)
        at org.apache.logging.log4j.core.impl.Log4jContextFactory.getContext(Log
4jContextFactory.java:47)
        at org.apache.logging.log4j.LogManager.getContext(LogManager.java:196)
        at org.apache.logging.log4j.LogManager.getLogger(LogManager.java:599)
        at app.freerouting.logger.FRLogger.<clinit>(FRLogger.java:13)
        at app.freerouting.gui.MainApplication.main(MainApplication.java:34)
Caused by: java.lang.ClassNotFoundException: javax.management.MBeanRegistrationE
xception
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown So
urce)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(U
nknown Source)
        at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
        ... 10 more

2022-06-07 10:31:51.630 [main] INFO  Freerouting v1.6.0 (build-date: 2022-06-06)

2022-06-07 10:31:51.786 [main] INFO  Opening 'C:\Users\hp\Desktop\testauto\testa
uto.dsn'...

```
* [ ] Please list the detailed steps to reproduce the problem
Just load any dsn file exported from Kicad 5.1.12

* [ ] What would you expect freerouting to do in this situation?
Fix bug? if it is.

* [ ] Is fixing this issue critical for your work?

**Comment by andrasfuchs** [comment]
It looks like there is a missing Chinese translation for one of the strings. I'll take a look, hopefully this is a quick fix.

Meanwhile you can try to run freerouting with the `-l en` command line argument, that should be ok.

**Comment by andrasfuchs** [comment]
It's fixed now in [the v1.6.1 release](https://github.com/freerouting/freerouting/releases/tag/v1.6.1). 

Thank you for your report @cdhigh!
